lib/frontend/get-github-subscriptions.js

```diff
@@ -7,13 +7,14 @@ module.exports = async (req, res, next) => {
 
   const { github, client, isAdmin } = res.locals;
 
-  const { installationId } = req.params;
+  const { installationId } = req.query;
   try {
     const { data: { login } } = await github.users.getAuthenticated();
     // get the installation to see if the user is an admin of it
     const { data: installation } = await client.apps.getInstallation({ installation_id: installationId });
-    // get all subscriptions from the database for this installation ID
-    const subscriptions = await Subscription.getAllForInstallation(installationId);
+    // get all subscriptions from the database for this installation ID and githubHost
+    const githubHost = new URL(installation.html_url).hostname;
```

Comment:
validate "installation" object before it is getting used

Reply:
in case host is null / empty log the installation.html_url

Reply:
ensure we have added new test case for this hostname logic, we can take this additional test as next pr as well

Reply:
test cases i will take in next PR. I have fixed broken test cases. new test cases i will right later